products/views.py

```diff
@@ -86,3 +84,25 @@ def get(self, request):
         except Product.DoesNotExist:
             return JsonResponse({'message': 'product_not_exist'}, status=400)
 
+from django.http import JsonResponse
+from django.views import View
+
+from .models import *
+
+class ProductDetailView(View) :
+    def get(self, request, product_id) :
+        product            = Product.objects.get(id = product_id)
+        images             = Image.objects.filter(product_id = product_id)
+        
+        result = {
+            'thumbnail_image_url': product.thumbnail_image_url,
+            'name'               : product.name,
+            'price'              : product.price,
+            'description'        : product.description,
+            'size'               : [value.name for value in Size.objects.all()],
+            'color'              : [value.name for value in Color.objects.all()],
```

Comment:
정확한 이름으로 작성하는 습관을 들여주세요!
```suggestion
            'size'               : [size.name for size in Size.objects.all()],
            'color'              : [color.name for color in Color.objects.all()],
```

Reply:
네! 염두하겠습니다!